webapp/channels/src/components/widgets/users/avatar/avatar.tsx

```diff
@@ -27,11 +27,11 @@ export const getAvatarWidth = (size: TAvatarSizeToken) => {
         return 36;
     case 'xl':
         return 50;
-    case 'xl-custom-DM':
-        return 96;
-    case 'xl-custom-GM':
-        return 72;
     case 'xxl':
```

Comment:
Thanks for giving this a shot, but we can't actually change the existing sizes here because this component is exposed to plugins, and changing the meaning of `xxl` will mean that any plugins that use that size of avatar will be affected

Reply:
Ok, I guess I'll just switch back for now then.